app/views/user_mailer/in_person_verified.html.erb

```diff
@@ -10,7 +10,7 @@
   <%= t('user_mailer.in_person_verified.greeting') %><br>
   <%= t(
         'user_mailer.in_person_verified.intro',
-        location: @presenter.location_name,
+        location: @presenter.visited_location_name,
```

Comment:
[non-blocking] just curious when there is no location will the email display 'none' as the post office name? Meaning, would the sentence read something like `You successfully verified your identity at the none Post Office`? I don't think this is a scenario that would happen realistically but just curious if it _could_ happen.

Reply:
The email won't display 'none' as the post office name.  We don't have 'none' set as a default argument for `visited_location_name`.  I did test out this scenario with the below code change in the user mailer preview
```
  def in_person_verified
    UserMailer.with(user: user, email_address: email_address_record).in_person_verified(
      enrollment: in_person_enrollment_id_ipp,
      visited_location_name: nil,
    )
  end
```

and here is what's displayed:

<details>
<summary>in_person_verified email with a nil visited_location_name:</summary>

![InPersonVerifiedNilVisitedLocationName](https://github.com/user-attachments/assets/3ac934c9-4922-4b87-be2d-40057cd17205)

</details>